govc/events/command.go

```diff
@@ -149,7 +151,7 @@ func (r *record) Write(w io.Writer) error {
 	if r.Type != "" {
 		kind = fmt.Sprintf(" [%s]", r.Type)
 	}
-	_, err := fmt.Fprintf(w, "[%s] [%s]%s %s\n", when, r.Category, kind, r.Message)
+	_, err := fmt.Fprintf(w, "[%s] [%s] [%d]%s %s\n", when, r.Category, r.Key, kind, r.Message)
```

Comment:
Thanks for your review.
Since kind is an empty string or a string starting with space, so space is not needed here.

https://github.com/vmware/govmomi/blob/f7bbf3db22758f787c5825bd75fcfd7c81cd5fa3/govc/events/command.go#L148-L151